hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -294,14 +290,6 @@ def write_s3_text(txt_content: io.StringIO, bucket, key: str) -> None:
 if __name__ == "__main__":
     logging.basicConfig(level=logging.INFO)
     FetcherConfig.get.cache_clear()  # Just in case
-    os.environ.setdefault(
-        "THREAT_EXCHANGE_DATA_BUCKET_NAME",
-        "jeberl-hashing-data20210324205948477200000003",
-    )
-    os.environ.setdefault("THREAT_EXCHANGE_PDQ_DATA_KEY", "threat_exchange_data/pdq.te")
-    os.environ.setdefault(
-        "THREAT_EXCHANGE_CONFIG_DYNAMODB", "jeberl-ThreatExchangeConfig"
-    )
```

Comment:
Why this change?

Reply:
@BarrettOlson and I discussed this but if feels weird to have hard coded data specific to my instance in the code. Dont necessarily think it's a security risk just seems off.

I will admit its very helpful for testing though. Wish we has some way to set these variables locally after terraform runs

Reply:
We could have local environment variables that was given as an optional override. Probably not that tricky to do with a shell or python script.